manifests/init.pp

```diff
@@ -421,6 +422,10 @@
     fail('ssh::sshd_config_banner must be set to be able to use sshd_banner_content.')
   }
 
+  if $ssh_gssapiauthentication != undef {
```

Comment:
This happens to sshd_gssapiauthentication too. I think I should remove the if statement for both....